### ISSUE TITLE
Assert when Mouse is on Overlay

### ISSUE THREAD

**Issue by DavidMansolino** [cyberbotics/webots#1743]
**Describe the Bug**
Moving the mouse on an overlay is raising an assert on develop:

```
webots-bin: gui/WbConsole.cpp:702: void WbConsole::appendLog(WbLog::Level, const QString&, bool, const QString&): Assertion `!logName.isEmpty()' failed.
./webots: line 88:  4778 Aborted                 (core dumped) "$webots_home/bin/webots-bin" "$@
```

**Comment by wangwangwsf** [comment]
How to control the movement of pioneer 3DX in webots?

**Comment by DavidMansolino** [comment]
Fixed in #1747
